### ISSUE TITLE
Clean up prometheus charts issues and PRs in stable repo

### ISSUE THREAD

**Issue by scottrigby** [prometheus-community/helm-charts#29]
After #11  and #28 

Note we're already holding PRs in stable (#13)

**Issue options:**
1. Close them all with a note to re-open if relevant in the new location (easiest on us, maybe not as nice for end users)
1. ~Automate this for users by transferring the issues~
    1. ~temporarily transfer repo to the [helm](https://github.com/helm/) org (because you can only move between repos [within the same org](https://docs.github.com/en/github/managing-your-work-on-github/transferring-an-issue-to-another-repository))~
    1. ~move issues from `helm/charts` to `helm/prometheus-helm-charts`~
    1. ~immediately transfer repo from `helm/prometheus-helm-charts` to `prometheus-community/helm-charts`~

Since the repo has already been transferred we can only do option 1 above.

**PR options:**

AFAIK there's no nice way for us to automate transferring open PRs from one repo to another and still allow the original PR author to own the PR. So I think our best option is to close open PRs in `helm/charts` with a friendly note on how they can do re-open the PR themselves (1. add new remote locally, 2. open PR from the same branch the initially did).

**Comment by monotek** [comment]
Do we have an ETA for this? I'm still on vacation until 7. September. I try to not use the laptop and i'm only on mobile until then. Hopefully it's enough to check issues later.

**Comment by scottrigby** [comment]
Yes the person who needs to give final approval for #11 is on vacation until this coming week

**Comment by scottrigby** [comment]
#11 is finished. We just need to finish #28 and then we can do this.

**Comment by gkarthiks** [comment]
I feel like doing the 2nd option is so tedious. Rather we can do one of the following.

* Following your options 1
* Ask the corresponding chart maintainers to take care of the issues and PRs in the helm/charts against their charts

**Comment by scottrigby** [comment]
@gkarthiks I agree, and we have already transferred the repo so we can only do option 1 now anyway 😄

**Comment by gkarthiks** [comment]
@scottrigby So I am going to take a stab on the `helm/charts` issue ( may be PR as well) and going to comment as following later today.

<hr/>
Hi `uname` the `stable/chart` chart has been deprecated effectively as per this PR `deprecation_PR_in_helm/charts` on Sep 5, 2020 and moved to https://github.com/prometheus-community/helm-charts.
<br/><br/>
Please close this Issue and re-open against the prometheus-community/helm-charts repo. Here's a friendly link for [new issue](https://github.com/prometheus-community/helm-charts/issues/new)
<br/><br/>
We hope to see you there. 👋 
<hr/>

does this sounds good?

**Comment by scottrigby** [comment]
Sounds great 👍 if you get to it before each chart maintainer I'm sure they'd appreciate the extra help. Someone will have to them close those issues (I wouldn't rely on each user to do so) but that's a bit less work. Thanks!

**Comment by gkarthiks** [comment]
@scottrigby @torstenwalter I have commented in the below linked issues under `helm/charts` so far. The left out issues in that repository are against `prometheus-operator` which we haven't completed yet. 

https://github.com/helm/charts/issues/951
https://github.com/helm/charts/issues/23718
https://github.com/helm/charts/issues/23674
https://github.com/helm/charts/issues/23649
https://github.com/helm/charts/issues/23582
https://github.com/helm/charts/issues/23551
https://github.com/helm/charts/issues/23388

https://github.com/helm/charts/issues/23547
https://github.com/helm/charts/issues/23518

https://github.com/helm/charts/issues/23535

https://github.com/helm/charts/issues/23711


Do we expect a time line and close those issues by ourself? Like say after 15 business days?

**Comment by scottrigby** [comment]
@gkarthiks Thanks! and yes either someone who is able (chart or repo maintainer, or the OP) can close those issues, or we can wait for stalebot to do it for us – or if not done by Nov 13th we will close all open issues when the that repo is no longer supported.

I will update https://github.com/helm/charts/issues/21103 as part of #28

**Comment by scottrigby** [comment]
https://github.com/helm/charts/issues/21103 updated

**Comment by scottrigby** [comment]
There is only one left, so let's leave this open until #1 is complete

**Comment by scottrigby** [comment]
OK, all PRs and issues related to stable prometheus charts are closed with a message pointing users here ✅
